index.js

```diff
@@ -0,0 +1,108 @@
+/*1-Write a console.log statement saying "Hello World!" for each language that you know.*/
+console.log('Halo, dunia!'); //Indoniasia
+console.log('Ciao, mondo!'); //Italian
+console.log('Hola, mundo!'); //Spanish
+
+//2-onsider the following code://
+console.log("I'm awesome");
+//Declare a variable age and initialize it with an integer, using these exact steps://
+var age;
+console.log('The Value of Age will be: undefined');
+console.log(age);
+age= 28;
+console.log('The Value of Age will be: 28');
+console.log(age);
+//4-Declare a variable name and assign a string to it.//
+var name = 'Hamam Alsamel';
+console.log('The Name will be: Hamam Alsamel');
+console.log(name);
+var x= name;
+console.log('The Value of X will be: Hamam Alsamel');
+console.log(x);
+//5-How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+var number= 7.25;
+console.log(number);
+var z= 7;
+console.log(z);
+if (number>=z){
+  console.log(number);
+  var newvalue = number;
+  console.log(newvalue);
+}
+//6- Arrays//
+
+const list = ['','',''];
+console.log('The array has no value becauswe its empty');
+console.log(list);
+
+const favouritanimal = ['Cat','Dog','Birds','elephant'];
```

Comment:
I think the instructions said to use a plural name for your array since it contains a list of items, so a better name would be `favouriteAnimals`